Dockerfile

```diff
@@ -0,0 +1,13 @@
+
+FROM keymetrics/pm2:latest-slim
```

Comment:
Ooops - there's no need to use the keymetrics image. I'll switch back to node:14.